app/api/datastore/entities/v1/base_application_entity.rb

```diff
@@ -11,7 +11,6 @@ class BaseApplicationEntity < Grape::Entity
         expose :provider_details
         expose :client_details
         expose :case_details
-        expose :interests_of_justice
```

Comment:
I may feel inclined to leave the base entity untouched and just add to the maat entity:

`unexpose : interests_of_justice`

Feels to me more explicit that way. But functionally would be the same.